src/main/java/ca/wolfram/beta/symath/operations/SubtractNode.java

```diff
@@ -0,0 +1,28 @@
+package ca.wolfram.beta.symath.operations;
+
+import ca.wolfram.beta.symath.MathNode;
+import ca.wolfram.beta.symath.base.BaseNode;
+
+/**
+ * Created by Allan Wang on 2017-05-11.
+ */
+public class SubtractNode {
+
+    private SubtractNode() {
+    }
+
+    /**
+     * Takes in a single node to subtract (minuend == 0)
+     *
+     * @param nodes subtrahend
+     * @return resulting node
```

Comment:
This will throw if arguements is incorrect because of NegateNode taking in only 1 arguement